gallery/src/components/NxTag/NxSelectableTagExample.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { NxSelectableTag } from '@sonatype/react-shared-components';
+
+function NxSelectableTagExample() {
+  const [tagSelected1, setTagSelected1] = useState(false),
+      onTagSelect1 = () => setTagSelected1(!tagSelected1),
+      [tagSelected2, setTagSelected2] = useState(false),
+      onTagSelect2 = () => setTagSelected2(!tagSelected2),
+      [tagSelected3, setTagSelected3] = useState(false),
+      onTagSelect3 = () => setTagSelected3(!tagSelected3),
+      [tagSelected4, setTagSelected4] = useState(false),
+      onTagSelect4 = () => setTagSelected4(!tagSelected4),
+      [tagSelected5, setTagSelected5] = useState(false),
+      onTagSelect5 = () => setTagSelected5(!tagSelected5),
+      [tagSelected6, setTagSelected6] = useState(false),
+      onTagSelect6 = () => setTagSelected6(!tagSelected6),
+      [tagSelected7, setTagSelected7] = useState(false),
+      onTagSelect7 = () => setTagSelected7(!tagSelected7),
+      [tagSelected8, setTagSelected8] = useState(false),
+      onTagSelect8 = () => setTagSelected8(!tagSelected8),
+      [tagSelected9, setTagSelected9] = useState(false),
+      onTagSelect9 = () => setTagSelected9(!tagSelected9),
+      [tagSelected10, setTagSelected10] = useState(false),
+      onTagSelect10 = () => setTagSelected10(!tagSelected10);
+
+  return (
+    <div>
```

Comment:
Could probably be a fragment

Reply:
> Could probably be a fragment

Yeah, I had the idea of building a tag container as well, but decided not to, replaced with a fragment here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/816141f59b568c0155da5c781552a94a5ab278b0